sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -439,7 +416,7 @@ __ESIMD_INTRIN __ESIMD_raw_vec_t(T, SZ)
   return retv;
 }
 
-inline constexpr __ESIMD_NS::uint
+inline __ESIMD_NS::uint
```

Comment:
why this change? this may lead to inefficiency or inability to use it in template meta-programming.

Note: you may change this to `if constexpr(...)` style to avoid evaluation of `false` branches (and potential related errors)

Reply:
This change is removed.